
WARNING from init.py: found 4 bidirectional edges, here is the first few:
('ROB', 'MARA')
('CRP', 'FIS')
('MARA', 'ROB')
('FIS', 'CRP')


WARNING from init.py: found 34 self-loops, here is one of them: ('OXYR', 'OXYR')

,
'Bacteria-RegulonDB':{
		'deg':[1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 17, 19, 20, 22, 25, 28, 29, 31, 34, 40, 41, 43, 46, 97, 127, 163],
		'freq':[0.512249443207127, 0.25389755011135856, 0.08463251670378619, 0.043429844097995544, 0.022271714922048998, 0.012249443207126948, 0.0033407572383073497, 0.005567928730512249, 0.0077951002227171495, 0.005567928730512249, 0.008908685968819599, 0.0022271714922048997, 0.005567928730512249, 0.0033407572383073497, 0.0033407572383073497, 0.0011135857461024498, 0.0011135857461024498, 0.0033407572383073497, 0.0011135857461024498, 0.0011135857461024498, 0.0011135857461024498, 0.0033407572383073497, 0.0011135857461024498, 0.0011135857461024498, 0.0022271714922048997, 0.0022271714922048997, 0.0011135857461024498, 0.0011135857461024498, 0.0022271714922048997, 0.0011135857461024498, 0.0011135857461024498],
		'edge2node':1.6492204899777283,
		'node2edge':0.6063470627954085,
		'edge2node_adj':adj_largestC()['Bacteria'][1],
		'node2edge_adj':adj_largestC()['Bacteria'][0],
		'edges':1481,
		'nodes':898
}
